### PR TITLE
Swift 3 update

### PR DESCRIPTION
**Pull request by brockboland**
Unfortunately, this includes a lot of pod update garbage in the Swift sample project. Sorry about that.

@vokal/ios-developers can I get some review please?

### CODE REVIEW

**Comment by bryanluby** [comment]
Sure

File: SwiftSampleProject/Podfile [hunk 1]
```diff
@@ -2,12 +2,10 @@ source 'https://github.com/CocoaPods/Specs.git'
...
-
-target 'SwiftyVokoder', :exclusive => true do
+target 'SwiftyVokoder' do
   pod 'Vokoder/Swift', :path => '../'
-  pod 'xUnique', :git => 'https://github.com/vokal/xUnique.git', :commit => '26adc7d50b'
-end
+  pod 'xUnique', :git => 'https://github.com/vokal/xUnique.git', :commit => '43d39de'
 
-target 'SwiftyVokoderTests', :exclusive => true do
-  pod 'Vokoder/Swift', :path => '../'
```
Comment (bryanluby) [inline SwiftSampleProject/Podfile]: Is this not needed for the test target now

File: SwiftSampleProject/SwiftyVokoder/Models/Station.swift [hunk 2]
```diff
@@ -56,17 +56,17 @@ extension Station: VOKMappableModel {
         return { (_, inputObject: NSManagedObject) in
             guard let
```
Comment (bryanluby) [inline SwiftSampleProject/SwiftyVokoder/Models/Station.swift]: With Swift 3 syntax I wonder if we should be bringing the `let` down now for better alignment?

File: SwiftSampleProject/SwiftyVokoder/Models/Stop.swift [hunk 3]
```diff
@@ -30,15 +30,15 @@ class Stop: NSManagedObject {
         init(value: String?) {
             if let
```
Comment (bryanluby) [inline SwiftSampleProject/SwiftyVokoder/Models/Stop.swift]: Similar `let` syntax comment as above

File: SwiftSampleProject/SwiftyVokoderTests/ManagedObjectContextTests.swift [hunk 4]
```diff
@@ -19,32 +19,33 @@ class ManagedObjectContextTests: XCTestCase {
...
+        let countOfStations = self.manager.count(for: Station.self)
         XCTAssertGreaterThan(countOfStations, 0)
         
-        tempContext.performBlockAndWait {
-            self.manager.deleteAllObjectsOfClass(Station.self, context: tempContext)
+        tempContext.performAndWait {
+            self.manager.deleteAllObjects(of: Station.self, context: tempContext)
         }
-        self.manager.saveAndMergeWithMainContextAndWait(tempContext)
+        self.manager.saveAndMerge(withMainContextAndWait:tempContext)
```
Comment (bryanluby) [inline SwiftSampleProject/SwiftyVokoderTests/ManagedObjectContextTests.swift]: Need space after colon

File: SwiftSampleProject/SwiftyVokoder/Models/Station.swift (hunk 2 above)
Comment (brockboland) [inline SwiftSampleProject/SwiftyVokoder/Models/Station.swift]: We'll also need to update our coding standards for this, but I feel like I'm missing something: Xcode isn't syntax coloring the `let` statements the same

screen shot 2016-09-23 at 9 02 53 am

**Comment by brockboland** [comment]
Pushed some updates. I also bumped the travis image to use Xcode 8 so the tests in the Swift sample can run, but the other sample projects might fail: I haven't tested them locally yet. I'll update those in a separate PR if need be.

**Comment by brockboland** [comment]
`xctool` doesn't support Xcode 8 yet. It's in master but not released: https://github.com/facebook/xctool/issues/704

It might be a good time to switch this project over to `fastlane`, though I've never done so with multiple projects in subdirectories, so that might be a project for another PR.

File: SwiftSampleProject/SwiftyVokoder/Models/Station.swift (hunk 2 above)
Comment (bryanluby) [inline SwiftSampleProject/SwiftyVokoder/Models/Station.swift]: That may be an Xcode issue

**Comment by bryanluby** [comment]
LGTM

**Comment by brockboland** [comment]
I'm going to merge this and finish cleanup in another PR.

**Pull request merged**